blackbox/docs/general/dml.rst

```diff
@@ -391,7 +391,8 @@ Import and Export
 Importing Data
 --------------
 
-Using the ``COPY FROM`` SQL statement, data can be imported into CrateDB.
+Using the ``COPY FROM`` statement, CrateDB can import data from files available
+over the network or files that are local to the CrateDB nodes.
```

Comment:
suggestion: CrateDB can import data from files which are accessible by each CrateDB node.
Also not perfect, but above does not highlight the the "network" files must also be accessible by each node, same as a local file must be accessible.